lte/gateway/c/core/oai/test/mock_tasks/BUILD.bazel

```
@@ -0,0 +1,38 @@
+# Copyright 2022 The Magma Authors.
+
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("@rules_cc//cc:defs.bzl", "cc_library")
+
+package(default_visibility = ["//lte/gateway/c/core:__subpackages__"])
```

```suggestion
package(default_visibility = ["//lte/gateway/c/core/test:__subpackages__"])
``` 
? or is it used outside of test